cpu/native/include/native_internal.h

```diff
@@ -10,6 +10,16 @@
  * Native CPU internal declarations
  */
 
+/**
+ * @defgroup    cpu_native_stdio    STDIO for native
+ * @ingroup     sys_stdio
+ * @brief       Standard input/output backend for native
+ *
+ * This will hook up RIOT's stdio to the host's stdio fds
```

Comment:
```suggestion
 * This will hook up RIOT's stdio to the host's stdio fds. It is the default
 * stdio implementation of the board `native`.
```

Since the other backends provide a description on how to enable the module, it might make sense to point out this doesn't need enabling.